deploy/registration-service.yaml

```diff
@@ -228,6 +238,8 @@ objects:
       verification.excluded_email_domains: ${VERIFICATION_EXCLUDED_EMAIL_DOMAINS}
       verification.message_template: ${VERIFICATION_MESSAGE_TEMPLATE}
       twilio.from_number: ${TWILIO_FROM_NUMBER}
+      woopra.domain: ${WOOPRA_DOMAIN}
+      segment-write-key: ${SEGMENT_WRITE_KEY}
```

Comment:
The same here.
In addition, the `woopra.domain` even doesn't match with the one that is used at line [178](https://github.com/codeready-toolchain/registration-service/pull/148/files#diff-d34232b191f95b876219f0dc263f2debac6ca94abf4535dd1433546b01a4a8baR178)

Reply:
updated here: fd6538e88bf12f20ad946a8f8d04acebd5acbb02